src/config.h

```diff
@@ -148,7 +148,7 @@ sds RSConfig_GetInfoString(const RSConfig *config);
 #define CONCURRENT_INDEX_MAX_POOL_SIZE 200  // Maximum number of threads to create
 #define GC_SCANSIZE 100
 #define DEFAULT_MIN_PHONETIC_TERM_LEN 3
-#define DEFAULT_FORK_GC_RUN_INTERVAL 10
+#define DEFAULT_FORK_GC_RUN_INTERVAL 100
```

Comment:
This should be 30 seconds (i.e 30) I have not idea when it was changed to 10. The value that we talked about that need to be changed to 100 is DEFAULT_FORK_GC_CLEAN_THRESHOLD